src/engine/filters/enginefiltermoogladder4.h

```diff
@@ -96,8 +96,9 @@ class EngineFilterMoogLadderBase : public EngineObjectConstIn {
         if (MODE == MoogMode::HighPassOversampling || MODE == MoogMode::HighPass) {
             m_postGainNew = 1;
         } else {
-            m_postGainNew = (1 + resonance / 4 * (1.1f + cutoff / sampleRate * 3.5f))
-                    * (2 - (1.0f - resonance / 4) * (1.0f - resonance / 4));
+            // See https://github.com/mixxxdj/mixxx/pull/11177 for the Jupyter
+            // notebook used to derive this
+            m_postGainNew = 1.0001784074555027f + (resonance * 0.9331585678097162f);
```

Comment:
I know. I kept the extra precision since the compiler truncates this anyways, and if for whatever reason you ever want to switch over to doubles you don't need to recompute the value. The definition of `kPi` in the same source file also has excess decimals for a 32-bit floating point number.